src/components/Input/Input.styles.ts

```diff
@@ -100,55 +100,22 @@ const StyledInput = styled.input<{
   outline: none;
 
   &:focus {
-    border-color:${error ? theme.secondary.red[500] : theme.primary.base.brand};
-    border-width:1px;
-    padding-top: ${
-      overridePadding
```

Comment:
will override padding functionality be gone?

Reply:
Nope all good, that's still there on the input